package j10_exception;

public class Ex04_MessageNaN {

	public static void main(String[] args) {

		// ** by Zero ( 나누기 0 )
		// => 정수형 연산 : ArithmeticException
		// => 실수형 연산 : Exception 이 일어나지 않음
		// Wrapper 클래스의 메서드 활용

		try {
			double d1 = 1.5 / 0.0;
			double d2 = 1.5 % 0.0;

			System.out.printf("실수 by zero Test : d1 = %f, d2 = %f", d1, d2);
		} catch (Exception e) {

		} finally {

		}

	}// main

}// class